src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -227,6 +228,33 @@ void UserInfoPopup::installEvents()
                                           });
             }
         });
+
+    // ignore highlights
+    QObject::connect(
+        this->ui_.ignoreHighlights, &QCheckBox::clicked, [this](bool gotClicked) mutable {
+            this->ui_.ignoreHighlights->setEnabled(false);
+
+            if (gotClicked) {
+                getApp()->highlights->blacklistedUsers.insertItem(
+                    HighlightBlacklistUser{this->userName_, false});
+                this->ui_.ignoreHighlights->setEnabled(true);
+            } else {
+                const auto &ignoreHighlightsUsersVector =

```

Comment:
Use a shorter name for this temporary such as `vector` since this is a temporary reference and only used to make the call to `getApp()->highlights->blacklistedUsers.getVector()` shorter.